scripts/release.js

```diff
@@ -156,6 +156,15 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '../..'));
+spawnOrFail('npx', ['create-react-app test-app']);
+process.chdir(path.join(__dirname, '../../test-app'));
+spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion}`]);
```

Comment:
We need to install the component library too right? And the hard part is how we validate that there is no error. This does not include any validation steps.